### ISSUE TITLE
Linux Server

### ISSUE THREAD

**Issue by Draakoor** [auroramod/h1-mod#798]
Hi there,

I am trying to run a server on linux/docker with wine.
After executing "wine h1-mod.exe -nosteam -dedicated +set net_port {{SERVER_PORT}} +exec server.cfg +map_rotate -memoryfix"
h1mod crashes.
I attached a crash/dump.
[h1-mod-crash-3-2024-03-05-21-51-01.zip](https://github.com/auroramod/h1-mod/files/14501874/h1-mod-crash-3-2024-03-05-21-51-01.zip)

**Comment by troopaofficial** [comment]
Did you get it working? Currently trying to make my own dedicated server with bots and have no clue where to start.

**Comment by Draakoor** [comment]
> Did you get it working? Currently trying to make my own dedicated server with bots and have no clue where to start.

currently not working.

**Comment by mjkzy** [comment]
working on latest build?

**Comment by warkingous** [comment]
is anyone able to run this using wine? it always crashes for me after loading common_mp

**Comment by mjkzy** [comment]
> is anyone able to run this using wine? it always crashes for me after loading common_mp

are you on latest? do you get a crashdump at all?

**Comment by warkingous** [comment]
> > is anyone able to run this using wine? it always crashes for me after loading common_mp
> 
> are you on latest develop? do you get a crashdump at all?

yes latest, it doesnt create crashdump, can u add me on discord? warkingous

**Comment by troopaofficial** [comment]
any updates on this?

**Comment by mjkzy** [comment]
Using Wine on h1-mod and/or iw7-mod can lead to weird issues happening. While i've noticed Wine to work using `winetricks` and then installing and logging into Steam, it still provides weird issues using solely Wine.

Even though its inconvenient, Proton runs the game and dedicated server great without much more needing done. I'd refer to this [IW7-Mod Linux guide] and try it for h1-mod.
